<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(0, 85, 9, 255), stop:1 rgba(255, 147, 147, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Ventricle Pacing Mode(VOO)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Mode Parameters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="connected">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;; color:rgb(0, 255, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="disconnected">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;; color:rgb(255, 0, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pacemaker_number">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;; color:rgb(255,255,255)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Lower Rate Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>275</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Upper Rate Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Ventricle Amplitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>425</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Ventricle Pulse Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>100</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string>Updated</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-13</x>
      <y>90</y>
      <width>1321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>-10</y>
      <width>21</width>
      <height>861</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyChanges">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>730</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(0,0,0);
background-color: rgb(255, 255, 216);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Apply Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetChanges">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>730</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(0,0,0);
background-color: rgb(255, 216, 216);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Reset Changes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>705</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>I have reviewed my changes </string>
    </property>
   </widget>
   <widget class="QPushButton" name="aai">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>240</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(0,0,0);
background-color: rgb(255, 255, 216);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>AAI Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="LRL_Current">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>210</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="URL_Current">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>285</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="VV_Current">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>360</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="VPW_Current">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>435</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aoo">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>140</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(0,0,0);
background-color: rgb(255, 255, 216);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>AOO Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>55</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 6px;

color: rgb(255,255,255);
background-color: rgb(155, 155, 155);
font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;; color:rgb(255, 255, 255)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="account">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 6px;

color: rgb(255,255,255);
background-color: rgb(0, 100, 255);
font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="voo">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>340</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(0,0,0);
background-color: rgb(255, 255, 216);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>VOO Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vvi">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>440</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(0,0,0);
background-color: rgb(255, 255, 216);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>VVI Mode</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LRL">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>210</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="URL">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>285</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VV">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>360</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VPW">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>435</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="INVALID">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;; color:rgb(255, 0, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="INVALID_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>285</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;; color:rgb(255, 0, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="INVALID_3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>360</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;; color:rgb(255, 0, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="INVALID_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>435</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;; color:rgb(255, 0, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>690</y>
      <width>981</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="update_pm">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>730</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(255,255,255);
background-color: rgb(0, 155, 0);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Upload to Pacemaker</string>
    </property>
   </widget>
   <widget class="QLabel" name="INVALID_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>700</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;; color:rgb(255, 0, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>730</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: rgb(0,0,0);
background-color: rgb(255, 216, 216);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string> Back to Main Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
